onnxscript/test/function_libs/torch_aten/ops_correctness_test.py

```diff
@@ -197,6 +197,13 @@ def _logcumsumexp_input_wrangler(
     return args, kwargs
 
 
+def _log_softmax_input_wrangler(
+    args: list[Any], kwargs: dict[str, Any]
+) -> tuple[list[Any], dict[str, Any]]:
+    kwargs["dim"] = args.pop()
```

Comment:
Why is this needed? Do we need dim as a positional argument then instead?

Reply:
dim is input, but in op it is a attribute.